### ISSUE TITLE
Add documentation for invoke

### ISSUE THREAD

**Issue by briggySmalls** [briggySmalls/cookiecutter-pypackage#4]
I really like using invoke for linting/formatting/testing etc. but people may not realise how to use it.

This should be added to the project documentation

**Comment by yngtodd** [comment]
I am curious about this. Currently when I run `invoke --release` on a new project, I am running into a missing  file error for `.coverage`:
```
FileNotFoundError: [Errno 2] No such file or directory: '/Users/ygx/src/rust/ferris/.coverage'
```
Are there extra args for `invoke coverage` that create this file, or does it need to be created manually?

**Comment by briggySmalls** [comment]
@yngtodd thanks so much for your message. You found a bug 🙈 

I've just pushed a fix, so future projects should not have this issue.

The problem is in the `clean_tests` function of `tasks.py`. The code will throw errors if the directories it is cleaning are not present.

You can see the fix I pushed in c6ac55541c62f75850af1080053cb984e5342005. If you are using python 3.8 then you can just add the `missing_ok=True` parameter to the call to unlink. But I've pushed a fix that supports all versions of python.

Sorry for the confusion, and thanks for using the tool
